### ISSUE TITLE
Medallion of Water Breathing Doesn't Work (Forge 1.20.1)

### ISSUE THREAD

**Issue by cartooneric84** [kwpugh/gobber_forge#83]
I'm not sure if I'm using it correctly, but isn't the Medallion of Water Breathing suppose to keep my air supply from going down and getting me damaged from drowning?  Because it's not really doing anything from me trying it out.

**Comment by kwpugh** [comment]
post a link to your latest game log.   I'm guessing some other mod is conflicting.

**Comment by cartooneric84** [comment]
[latest.log](https://github.com/kwpugh/gobber_forge/files/14381050/latest.log)
Like this?

**Comment by kwpugh** [comment]
Yeah, that is a lot of mods.  I couldn't even begin to figure which one might be the issue.

You have to start removing mods one at a time and test until you find the one that breaks.

**Comment by cartooneric84** [comment]
Yeah.  I like a lot of fun mods.  =P

Though...do you think Artifacts might have an effect since there's an item that works the way the water breathing medallion does?

**Comment by kwpugh** [comment]
Artifacts is compiled with cardinal components which may be the problem

**Comment by cartooneric84** [comment]
Unfortunately I just tried removing Artifacts for now and retried having the medallion with me underwater.  Bubbles still drained and I still took drowning damage.  Bugger!

**Comment by kwpugh** [comment]
That medallion has no config options.

Give the test version a try and let me know if anything different.

[Uploading Gobber2-Forge-1.20.1-2.8.3.jar.zip…]()

**Comment by cartooneric84** [comment]
I got good news for you!  I had the medallion in my inventory, was underwater, bubble meter ran out, and nothing happened to me.  No drowning damage!  In other words, it worked!

**Comment by kwpugh** [comment]
wonderful.  I think the issue was mis-aligned parentheses in my code (yikes)

**Comment by cartooneric84** [comment]
> wonderful. I think the issue was mis-aligned parentheses in my code (yikes)

Life of a programmer.  If the code doesn't work, you ask why.  If the code DOES work....you still ask WHY! Lol!
